packages/bundler-plugin-core/src/types.ts

```diff
@@ -198,6 +198,17 @@ export type Options = Omit<IncludeEntry, "paths"> & {
    * Defaults to `false`.
    */
   injectReleasesMap?: boolean;
+
+  /**
+   * If set to true, the plugin will inject an additional `SENTRY_BUILD_INFO` variable.
+   * This contains information about the build, e.g. dependencies, node version and other useful data.
+   *
+   * This is still experimental and subject to change!
+   *
+   * Defaults to `false` for now.
+   * @hidden
+   */
+  injectBuildInformation?: boolean;
```

Comment:
l: Just a thought. Should we make this an experiment for now? Just so we don't "break" if we decide to unship? Iirc we are still not a 100% sure if we want to go this route.

Reply:
Yes, probably makes sense! I'll just make it `_experiments.injectBuildInformation` for now 👍